code/modules/chemistry/Reagents-Drugs.dm

```diff
@@ -467,6 +467,21 @@ datum
 					REMOVE_ATOM_PROPERTY(M, PROP_MOB_STAMINA_REGEN_BONUS, "caffeine_rush")
 				..()
 
+			calculate_depletion_rate(var/mob/affected_mob, var/mult = 1)
+				. = ..()
+				var/caffeine_amt = holder.get_reagent_amount(src.id)
+				switch(caffeine_amt) //use ~midpoints for depeletion rate thresholds - need stronger coffees or blends to overcaffeinate
+					if(0 to 3)
+						. *= 0.5
+					if(10 to 30)
+						. *= 2
+					if(30 to 50)
+						. *= 4
+					if(50 to INFINITY)
+						. *= 5
```

Comment:
This doesn't seem consistent with previous behavior, i.e. these values are not just a multiple of the previous ones. What gives?

Reply:
I am not a fan at all of just missing the 3 to 10 value in the switch statement. Without context it almost feels like an oversight/error. Would much prefer the baseline was set around the first batch (0 to 3) then the others scaled accordingly (increasingly).

Reply:
Agreed with Mordent on this one, although it seems consistent with how the code was written previously it'd probably be better the other way